python/revng/model/migrations/migrate.py

```diff
@@ -0,0 +1,41 @@
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+from revng.model import latest_version  # type: ignore
```

Comment:
Why `type: ignore`?

Reply:
(Note that code is edited, but the `# type: ignore` is still here)

This was decided [here](https://mattermost.rev.ng/revng/pl/wruqhojdypg6xeubruepzuzo4h). Are we okay with this, or should we invest time to get mypy working here?

Reply:
Do you mean remove the `# type: ignore`, or remove the import (and use `importlib` instead)?

Removing `# type: ignore` does fail with

```
error: Module "revng.model" has no attribute "Binary"  [attr-defined]
```

Reply:
OK, this is likely due to the fact that mypy does not see the generated code. Add a comment.

Reply:
Done.